### ISSUE TITLE
Ukidss.list_databases() fails

### ISSUE THREAD

**Issue by odysseus9672** [astropy/astroquery#1763]
When trying to use the `list_databases` method of `Ukidss` I get the following error.

```
In [1]: from astroquery.ukidss import Ukidss                                    
dbus[49296]: Dynamic session lookup supported but failed: launchd did not provide a socket path, verify that org.freedesktop.dbus-session.plist is loaded!
dbus[49296]: Dynamic session lookup supported but failed: launchd did not provide a socket path, verify that org.freedesktop.dbus-session.plist is loaded!

In [2]: Ukidss.list_databases()                                                 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-5e67702382f5> in <module>
----> 1 Ukidss.list_databases()

/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/astroquery/wfau/core.py in list_databases(self)
    674         List the databases available from the WFAU archive.
    675         """
--> 676         self.databases = set(self.all_databases + self._get_databases())
    677         return self.databases
    678 

TypeError: can only concatenate tuple (not "list") to tuple
```

Both the underlying `all_databases` member and `_get_databases` method are working. The problem appears to be that the `_get_databases` member is returning a list. This should be fixable by replacing line 676 of `wfau/core.py` with:
```
        self.databases = set(self.all_databases + tuple(self._get_databases()))
```
or
```
        self.databases = set(list(self.all_databases) + self._get_databases())
```
This is with Python 3.6 and 3.7 using astroquery 0.4 and 0.4.1.

**Comment by keflavich** [comment]
Yep.... this is indeed a bug.  If you want to submit this change as a pull request - ideally with some test coverage, it would be welcome!

**Comment by keflavich** [comment]
Closed by #1766
